resources/compass/charts/prometheus-postgres-exporter/values.yaml

```diff
@@ -0,0 +1,351 @@
+            description: "Maximum time spent in the statement, in milliseconds"
+        - mean_time_seconds:
+            usage: "GAUGE"
+            description: "Mean time spent in the statement, in milliseconds"
+        - stddev_time_seconds:
+            usage: "GAUGE"
+            description: "Population standard deviation of time spent in the statement, in milliseconds"
+        - rows:
+            usage: "COUNTER"
+            description: "Total number of rows retrieved or affected by the statement"
+        - shared_blks_hit:
+            usage: "COUNTER"
+            description: "Total number of shared block cache hits by the statement"
+        - shared_blks_read:
+            usage: "COUNTER"
+            description: "Total number of shared blocks read by the statement"
+        - shared_blks_dirtied:
+            usage: "COUNTER"
+            description: "Total number of shared blocks dirtied by the statement"
+        - shared_blks_written:
+            usage: "COUNTER"
+            description: "Total number of shared blocks written by the statement"
+        - local_blks_hit:
+            usage: "COUNTER"
+            description: "Total number of local block cache hits by the statement"
+        - local_blks_read:
+            usage: "COUNTER"
+            description: "Total number of local blocks read by the statement"
+        - local_blks_dirtied:
+            usage: "COUNTER"
+            description: "Total number of local blocks dirtied by the statement"
+        - local_blks_written:
+            usage: "COUNTER"
+            description: "Total number of local blocks written by the statement"
+        - temp_blks_read:
+            usage: "COUNTER"
+            description: "Total number of temp blocks read by the statement"
+        - temp_blks_written:
+            usage: "COUNTER"
+            description: "Total number of temp blocks written by the statement"
+        - blk_read_time_seconds:
+            usage: "COUNTER"
+            description: "Total time the statement spent reading blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)"
+        - blk_write_time_seconds:
+            usage: "COUNTER"
+            description: "Total time the statement spent writing blocks, in milliseconds (if track_io_timing is enabled, otherwise zero)"
+
+    pg_compass_director_tenants_total:
+      query: "SELECT COUNT(ID) as quantity FROM business_tenant_mappings;"
+      master: true
+      metrics:
+        - quantity:
+            usage: "COUNTER"
+            description: "Quantity of total tenants in table"
+
+    pg_compass_director_tenants_active:
+      query: "SELECT COUNT(ID) as quantity FROM business_tenant_mappings WHERE status='Active';"
+      master: true
+      metrics:
+        - quantity:
+            usage: "COUNTER"
+            description: "Quantity of active tenants in table"
+
+    pg_compass_director_tenants_with_at_least_one_runtime_or_application:
```

Comment:
Hmm this name is quite long. What do you think about `compass_director_used_tenants` or something like that?